UnitySDK/Assets/ML-Agents/Examples/GridWorld/Scenes/GridWorld.unity

```diff
@@ -728,110 +746,119 @@ BoxCollider:
 --- !u!33 &665184836
 MeshFilter:
   m_ObjectHideFlags: 0
-  m_PrefabParentObject: {fileID: 33825707296015202, guid: 5c2bd19e4bbda4991b74387ca5d28156,
-    type: 2}
-  m_PrefabInternal: {fileID: 0}
+  m_CorrespondingSourceObject: {fileID: 33825707296015202, guid: 5c2bd19e4bbda4991b74387ca5d28156,
+    type: 3}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 665184832}
   m_Mesh: {fileID: 10202, guid: 0000000000000000e000000000000000, type: 0}
 --- !u!1001 &673061833
-Prefab:
+PrefabInstance:
   m_ObjectHideFlags: 0
   serializedVersion: 2
   m_Modification:
     m_TransformParent: {fileID: 0}
     m_Modifications:
-    - target: {fileID: 4124767863011510, guid: 5c2bd19e4bbda4991b74387ca5d28156, type: 2}
+    - target: {fileID: 1488387672112076, guid: 5c2bd19e4bbda4991b74387ca5d28156, type: 3}
+      propertyPath: m_Name
+      value: FloatAgent
```

Comment:
Change one of the prefab instances to have a different name and compression type.